### ISSUE TITLE
Allow to use module attributes in assert_called

### ISSUE THREAD

**Issue by sascha-wolf** [appunite/mockery#31]
When I try to use module attributes in an `assert_called` match, I receive the following error:

```
== Compilation error in file test/authentication/storage/redis_test.exs ==
** (FunctionClauseError) no function clause matching in Kernel.@/1    
    (elixir) expanding macro: Kernel.@/1
```

Example for match:

```elixir
assert_called Redix, :command, [:redix, ["HSET", @namespace, "key", "value"]]
```

**Comment by amatalai** [comment]
Fixed in #34 

Patch will be released to hex in next few days. I'm sorry it took so long.

**Comment by amatalai** [comment]
It's released
